docs/DeveloperGuide.md

```diff
@@ -1,15 +1,126 @@
 # Developer Guide
+Welcome to the NUSPlanner Developer Guide!
+We hope this documents serves useful to understand the behind-the-scenes working of our product. 
+
+## Table of Contents
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}'
+
+## Getting started
+
+Refer to the user guide [here](https://github.com/AY2223S2-CS2113-F13-3/tp/blob/master/docs/UserGuide.md).
+
 
 ## Design & implementation
```

Comment:
Very clear explanation of the features! Would be helpful to have an explanation of _why_ you implemented your feature logic this way so other developers can understand your considerations.